components/prod/azure-public-dns.tf

```diff
@@ -0,0 +1,175 @@
+module "public-dns-1" {
```

Comment:
this should have the name of the zone in it

Reply:
Had chat with @naikajah about this before this initial PR - will implement addition of zone names and update

Reply:
@timja - agree with the stepped PR, 1 zone at a time. Are we going to be merging each zone addition to master as I believe we won't get the TF plan confirmation "Nothing to change" until all zones/recordsets are imported

Reply:
yes we should be merging each one to master 1 at a time, I see no reason we will see no change as long as we import each zone at a time?